src/test/java/com/easysplit/ess/shared/utils/TestUtils.java

```diff
@@ -0,0 +1,61 @@
+package com.easysplit.ess.shared.utils;
+
+import java.time.Instant;
+import java.util.Random;
+
+/**
+ * Class with utility methods for testing
+ */
+public class TestUtils {
+    /**
+     * Private constructor to prevent instantiation of the class.
+     * This class should only hold string constants and should not be instantiated
+     */
+    private TestUtils() {
+
```

Comment:
If a constructor is not used, should we keep this empty space as a body?

Reply:
In general, for this and the above comments, let's keep the empty constructors in only one line. I made the change for all current empty constructors